ALICE3/TableProducer/OTF/onTheFlyRICHPID.cxx

```diff
@@ -15,6 +15,9 @@
 
 #include <utility>
 #include <cmath>
+
+#include <TPDGCode.h>
```

Comment:
Sorry if I miss something - just for my own understanding, was the addition of this header needed? It seems it is the only change in this and the other `onTheFly` task?

Reply:
Ah - never mind. It is needed to define variables like `kElectron, kMuonMinus, kPiPlus, kKPlus, kProton` - they existed before but not anymore. Thanks!!!